docs/_config.yml

```diff
@@ -1,6 +1,6 @@
 highlighter: pygments
 markdown: kramdown
-gems:
+plugins:
```

Comment:
One thing I am worried though we replace this and some Jenkins builds don't work because the jekyll version is too old (looks it requires 3.5+?).

Reply:
Ah OK I'll back that out. Keeping jekyll up to date might be needed eventually but not worth it here.